### ISSUE TITLE
Phpgrid server

### ISSUE THREAD

**Issue by docwho12** [ft-/phpGridServer#13]
Hi FT!

I got the phpgridserver running  DB for grid is working but can't get run the arriba simulator. I use Debian 9 with php 7.0 the simulator starts and send xmlreader probs on linux??? xcache is not running on the php 7.0. and your Install file sorry looks like something is missing or I'm to stupid:-( can you give me some hints please - thanks!

**Comment by ft-** [comment]
did you use the INSTALL.md?

**Comment by Jam-Mc** [comment]
Good day

Advance information
I'm using Debian 8 with Opensim version 0.9.0.0 php 5

The world is online now I want to log in the viewer and it does not work

I also adapted the Apache2 I am a little bit surprised if this
Alias ​​/ login /var/www/html/index.php/login is correct.

utf-8 koodierung is also correct.

maybe I have misconfigured the inis in the opensim.

I ask for help that my advice is over

**Comment by ft-** [comment]
0.9 config and Arriba have slight differences
If you use a opensim.ini example just replace all ports with the one used for phpGridServer,

**Comment by docwho12** [comment]
Yes I have the same problems. I get all install and I used the install.md maybe it would be helpful if you write a install.md for us dummies. it would be great to get the php grid server to run - can I only use arriba or the new 0.91 too question over questions  think we need more input ;-)

**Comment by ft-** [comment]
I cannot remove the requirement to at least be able to deal with apache http server. Adjusting apache modules is pretty specific to the version used. Similar about php. I cannot write a manual for each and every OpenSim or OpenSim derived version.
I cannot target with it a person that does not know how to adjust apache configuration.

Interesting, a checklist seems to be too complex for people?
